evm_arithmetization/src/fixed_recursive_verifier.rs

```diff
@@ -1526,13 +1551,29 @@ where
                 nonzero_pis.insert(key, value);
             }
 
+            #[cfg(not(feature = "cdk_erigon"))]
+            let burn_addr_offset = 0;
+            #[cfg(feature = "cdk_erigon")]
+            let burn_addr_offset = BurnAddrTarget::SIZE;
```

Comment:
nit: for code like this I'd prefer something like
```suggestion
            let burn_addr_offset = match cfg!(feature = "cdk_erigon) {
                true => BurnAddrTarget::SIZE,
                false => 0
            };
```

It makes it easier to later factor out the feature, and makes the code more readable